src/app/projects/projects.component.ts

```diff
@@ -159,9 +160,16 @@ export class ProjectsComponent implements OnInit, OnDestroy {
   /**
    * Event handler called when filters component updates list of matching apps.
    */
-  public updateMatching() {
+  public updateMatching(source: string) {
     // map component gets filtered apps
     this.mapApps = this.filterApps.filter(a => a.isMatches);
+		// Prevent snackbar on initial page load
+		if (2 <= this.filterCount) {
```

Comment:
If someone loads the page, toggles a filter off, then activates all three filters, will they get feedback? i.e. is there an instance where they could be using three filters after page load?

Reply:
Oh yes, this isn't a limit to active filters. It prevents the snackbar from showing up when the page initially loads. Basically, it's saying wait until the third call to this code before showing the snackbar.